.github/workflows/ci.yml

```diff
@@ -0,0 +1,108 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: XNIO CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  build-all:
+    name: Compile (no tests) with JDK 8
+    runs-on: ubuntu-latest
+    steps:
+    - uses: n1hility/cancel-previous-runs@v2
+      with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository
+        key: m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+            m2-
+    - uses: actions/checkout@v2
+    - name: Host information
+      run: netstat -anp
+      shell: bash
+    - name: Set up JDK 8
+      uses: joschi/setup-jdk@v2
+      with:
+        java-version: 8
+    - name: Build
+      run: mvn -U -B -fae -DskipTests -Dfindbugs clean install
```

Comment:
AFAIK we don't have findbugs set up in XNIO.

Reply:
Sorry, missed that